ConnorAndersonLarson/PlanetParty
# Feature/edit text single planet

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Add state to `PlanetInfo` component, and add an input field to the gravity fact so that a user can input a weight on earth and see how much that would weigh on the respective planet they are viewing.

### What ticket(s) do the changes resolve?
#39 

Please Link Issues to this PR as necessary.

### Notes for the Reviewer

This is just basic styling, definitely okay to change as needed. 

### How to Test Changes?

Pull down this branch, should be no errors in the console when viewing the website locally. When clicking on a planet should see a single planet view with a default of 100 pounds on earth is  XX pounds on the respective planet. Users should be able to input whatever positive integer they wish in this field and see changes render instantly.

### Next Iterations
1. Make this view and input section responsive.
2. Test this input functionality.
3. Style this to match Katie's branch.
